impersonation-filter/src/test/scala/org/openrepose/filters/impersonation/IdentityResponses.scala

```diff
@@ -0,0 +1,321 @@
+package org.openrepose.filters
+
+
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+
+trait IdentityResponses {
+
+  protected final val VALID_TOKEN = "validToken"
+  protected final val VALID_USER = "validUser"
+
+  def tokenDateFormat(dateTime:DateTime): String = {
+    ISODateTimeFormat.dateTime().print(dateTime)
+  }
+
+  def adminAuthenticationTokenResponse(token: String = "glibglob",
+                                       expires: DateTime = DateTime.now()): String = {
+    val formattedTime = tokenDateFormat(expires)
+
+    s"""
```

Comment:
Cleaner code that's easier to read. Performance isn't going to be an issue in test code like this. 

You would have to parse the file, but that can be done in about one line of scala code. 

On October 9, 2015 11:17:45 AM CDT, Dimitry USHAKOV <notifications@github.com> wrote:
>> +import org.joda.time.format.ISODateTimeFormat
>> +
>> +trait IdentityResponses {
>> +
>> +  protected final val VALID_TOKEN = "validToken"
>> +  protected final val VALID_USER = "validUser"
>> +
>> +  def tokenDateFormat(dateTime:DateTime): String = {
>> +    ISODateTimeFormat.dateTime().print(dateTime)
>> +  }
>> +
>> +  def adminAuthenticationTokenResponse(token: String = "glibglob",
>> +                                       expires: DateTime =
>DateTime.now()): String = {
>> +    val formattedTime = tokenDateFormat(expires)
>> +
>> +    s"""
>
>But then wouldn't I have to parse the file to add my dynamic data? 
>What's the advantage of that over this?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/rackerlabs/impersonation-filter-bundle/pull/1/files#r41648661

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.